src/Domain/CampaignRepository.php

```diff
@@ -28,25 +28,27 @@ class CampaignRepository extends SalesforceReadProxyRepository
      */
     public function findRecentLiveAndPendingGiftAidApproval(): array
     {
-        $oneWeekAgo = (new DateTime('now'))->sub(new \DateInterval('P7D'));
-        $fourMonthsAgo = (new DateTime('now'))->sub(new \DateInterval('P4M'));
```

Comment:
we had the one week ago info repeated 5 times in this file in various forms. I took four of them out instead of updating all 5.